bunsen/bunsen-avro/src/main/java/com/cerner/bunsen/avro/converters/DefinitionToAvroVisitor.java

```diff
@@ -702,60 +892,13 @@ public HapiConverter<Schema> visitMultiValued(
   public HapiConverter<Schema> visitChoice(
       String elementName, Map<String, HapiConverter<Schema>> choiceTypes) {
 
-    List<Field> fields =
```

Comment:
I have not looked at the rest of the changes in this file yet (will do later); are we making any logic change here or is it just refactoring code that was here into the methods below?